go/tasks/pluginmachinery/flytek8s/testdata/imagepull-failurepod.json

```diff
@@ -0,0 +1,411 @@
+                        "value": "project"
+                    },
+                    {
+                        "name": "FLYTE_INTERNAL_EXECUTION_DOMAIN",
+                        "value": "development"
+                    },
+                    {
+                        "name": "FLYTE_INTERNAL_TASK_PROJECT",
+                        "value": "project"
+                    },
+                    {
+                        "name": "FLYTE_INTERNAL_TASK_DOMAIN",
+                        "value": "development"
+                    },
+                    {
+                        "name": "FLYTE_INTERNAL_TASK_NAME",
+                        "value": "common.library.utils.get_flyte_id"
+                    },
+                    {
+                        "name": "FLYTE_INTERNAL_TASK_VERSION",
+                        "value": "3c6869db7101c619908f2b568fa851a9df0016c2"
+                    },
+                    {
+                        "name": "FLYTE_INTERNAL_PROJECT",
+                        "value": "project"
+                    },
+                    {
+                        "name": "FLYTE_INTERNAL_DOMAIN",
+                        "value": "development"
+                    },
+                    {
+                        "name": "FLYTE_INTERNAL_NAME",
+                        "value": "common.library.utils.get_flyte_id"
+                    },
+                    {
+                        "name": "FLYTE_INTERNAL_VERSION",
+                        "value": "3c6869db7101c619908f2b568fa851a9df0016c2"
+                    },
+                    {
+                        "name": "AWS_RETRY_MODE",
+                        "value": "standard"
+                    },
+                    {
+                        "name": "AWS_METADATA_SERVICE_TIMEOUT",
+                        "value": "5"
+                    },
+                    {
+                        "name": "AWS_METADATA_SERVICE_NUM_ATTEMPTS",
+                        "value": "20"
+                    },
+                    {
+                        "name": "EMIT_CONTAINER_METRICS",
+                        "value": "true"
+                    },
+                    {
+                        "name": "FLYTE_STATSD_HOST",
+                        "value": "stats.statsagent"
+                    },
+                    {
+                        "name": "FLYTE_CREDENTIALS_AUTH_MODE",
+                        "value": "basic"
+                    },
+                    {
+                        "name": "FLYTE_CREDENTIALS_AUTHORIZATION_METADATA_KEY",
+                        "value": "flyte-authorization"
+                    },
+                    {
+                        "name": "FLYTE_CREDENTIALS_SCOPE",
+                        "value": "svc"
+                    },
+                    {
+                        "name": "FLYTE_CREDENTIALS_CLIENT_SECRET",
+                        "value": "E2EeKjJvwKngjrUnmyfvL6tnf8jcPGrVdAxQd8u0"
```

Comment:
can we clean up some of these things?

Reply:
i mean this is not really a secret, but still

Reply:
ohh yes